### ISSUE TITLE
(WCAG 2.1) Focus outline color does not meet minimum contrast ratio of 3:1 

### ISSUE THREAD

**Issue by dkatzz** [civiform/civiform#5533]
OBSERVED RESULTS:

- The change in focus indicator / focus outline color for the following buttons do not meet minimum contrast ratio of 3:1 (Foreground: ##000000 Background: #113F9F, Contrast Ratio: 2.23).
- The black outline is barely visible to the user.
- Apply buttons, Continue button, Previous button, Save and next button, Add household member
- ![Screenshot](https://github.com/civiform/civiform/assets/86739416/ac1b4823-3aa7-4
- ![Screenshot](https://github.com/civiform/civiform/assets/86739416/93edac91-286d-40f3-861c-bbc0dfdca4c0)
- ![Screenshot](https://github.com/civiform/civiform/assets/86739416/5e5aac3a-72dc-4437-9574-eea265ce4533)
- ![Screenshot](https://github.com/civiform/civiform/assets/86739416/5f5c2ff4-8d55-499b-a875-217b85af9d81)
- ![Screenshot](https://github.com/civiform/civiform/assets/86739416/3ca35c34-79dd-4dbc-9ddf-fec0cb31a801)
- ![Screenshot](https://github.com/civiform/civiform/assets/86739416/14740e86-51e5-42d2-b0cf-4a80100b2230)

EXPECTED RESULTS:

- The focus indicator outline should meet a minimum contrast ratio of 3:1 to the background. Keyboard focus indicator should be visually distinct..

STEPS TO REPRODUCE:

- Navigate to [URL](https://demo.civiform.dev/)
- Navigate to the ‘Apply’ links using keyboard TAB or SHIFT TAB key.
- Observe the focus outline color do not meet minimum contrast ratio of 3:1

ADDITIONAL INFORMATION:

- [ W3org Reference - 2.4.11: Focus Appearance (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-minimum.html)

ENVIRONMENT DETAILS:

- Device(s) (brand, OS version, test key or release key): Windows 10 Enterprise (64-bit),MacOS Ventura Version 13.4
- Screen reader name and version: JAWS 2022.2211.7, NVDA 2022.3, VoiceOver
- Browser name and version: Chrome Version 113.0.5672.93OBSERVED RESULTS:OBSERVED RESULTS:

**Comment by melanie-exygy** [comment]
Once #5662 is completed, our focus color will be #2491ff, which is the USWDS default.  This will give us a contrast ratio of 2.93 in respect to the dark blue buttons. 

![image](https://github.com/civiform/civiform/assets/134556723/21a84585-f01f-4a0f-b884-f84dd040b9bd)

To complete the above issue, I recommend changing the focus color setting in `_uswds-theme.scss` like this:

![image](https://github.com/civiform/civiform/assets/134556723/a86183f0-c3bb-451d-b478-302f12889720)

Rather than setting the color to green, as in the example above, we should set it to the color recommended by UX.  This color will need to be one that offers an acceptable contrast ratio with both the blue buttons and the white background.  @sijiayam, would you be able to add the desired color to this issue description?  Please also add any other desirable focus style changes such as width or offset.

**Comment by sijiayam** [comment]
@melanie-exygy My understanding is the blue background and the focus color are the only places where contrast is not 3:1. My recommendation would be changing the background color rather than the focus color. 

My suggesting is to use tailwind's background `bg-blue-900` or 'blue-900' or '#1e3a8a' for the homepage banner. I have checked that this will create a 3.24:1 contrast and is visually only a subtle difference. 

It also seems that the 'seattle-bg-blue' is not seattle's brand blue. We may want to look at renaming that in the future.

Note: issues regarding the contrast between the button blue and focus blue will be handled in issue https://github.com/civiform/civiform/issues/5532

**Comment by melanie-exygy** [comment]
> @melanie-exygy My understanding is the blue background and the focus color are the only places where contrast is not 3:1. My recommendation would be changing the background color rather than the focus color.
> 
> My suggesting is to use tailwind's background `bg-blue-900` or `blue-900` or `#1e3a8a` for the homepage banner. I have checked that this will create a 3.24:1 contrast and is visually only a subtle difference.
> 
> It also seems that the `seattle-bg-blue` is not seattle's brand blue... We may want to look at renaming that in the future.
> 
> Note: issues regarding the contrast between the button blue and focus blue will be handled in issue #5532

Thanks for looking at this @sijiayam .  I think this issue was for the button contrast, not #5532.  #5532 was for the focus color vs.  the white background of the forms.  I closed that issue after bringing in the USWDS focus because the contrast was acceptable.  We didn't have an issue for the contrast of the homepage banner, but it is definitely a good consideration.  Would it be possible to address both the button contrast and the contrast for the homepage banner here in issue #5533?
